eng/pipelines/prepare-pipelines.yml

```diff
@@ -0,0 +1,10 @@
+trigger: none
+
+extends:
+  template: /eng/common/pipelines/templates/steps/prepare-pipelines.yml
+  parameters:
+    Repository: $(Build.Repository.Name)
+    Prefix: cpp
+    CIConventionOptions: ''
+    UPConventionOptions: '--variablegroup 58 --variablegroup 76 --variablegroup 56'
+    TestsConventionOptions: ''
```

Comment:
Add a new line character to avoid `missing eof` warning.

Reply:
Done.